datasets/evidence_infer_treatment/evidence_infer_treatment.py

```diff
@@ -0,0 +1,231 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Inferring Which Medical Treatments Work from Reports of Clinical Trials"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import os
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{lehman-etal-2019-inferring,
+    title = "Inferring Which Medical Treatments Work from Reports of Clinical Trials",
+    author = "Lehman, Eric  and
+      DeYoung, Jay  and
+      Barzilay, Regina  and
+      Wallace, Byron C.",
+    booktitle = "Proceedings of the 2019 Conference of the North {A}merican Chapter of the Association for Computational Linguistics: Human Language Technologies, Volume 1 (Long and Short Papers)",
+    month = jun,
+    year = "2019",
+    address = "Minneapolis, Minnesota",
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/N19-1371",
+    pages = "3705--3717",
+}
+"""
+
+_DESCRIPTION = """\
+Data and code from our "Inferring Which Medical Treatments Work from Reports of Clinical Trials", NAACL 2019. This work concerns inferring the results reported in clinical trials from text.
+
+The dataset consists of biomedical articles describing randomized control trials (RCTs) that compare multiple treatments. Each of these articles will have multiple questions, or 'prompts' associated with them. These prompts will ask about the relationship between an intervention and comparator with respect to an outcome, as reported in the trial. For example, a prompt may ask about the reported effects of aspirin as compared to placebo on the duration of headaches. For the sake of this task, we assume that a particular article will report that the intervention of interest either significantly increased, significantly decreased or had significant effect on the outcome, relative to the comparator.
+
+The dataset could be used for automatic data extraction of the results of a given RCT. This would enable readers to discover the effectiveness of different treatments without needing to read the paper.
+"""
+
+
+class EvidenceInferenceConfig(datasets.BuilderConfig):
+    """ BuilderConfig for NewDataset"""
+
+    def __init__(self, zip_file, **kwargs):
+        """
+
+        Args:
+            zip_file: The location of zip file containing original data
+            **kwargs: keyword arguments forwarded to super.
+        """
+        self.zip_file = zip_file
+        super().__init__(**kwargs)
+
+
+class EvidenceInference(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class EvidenceInferTreatment(datasets.GeneratorBasedBuilder):
```
it should be the camel case version of the dataset script name `evidence_infer_treatment`